lib/src/components/NxCollapsibleItemsSelect/NxCollapsibleItemsMultiSelect/NxCollapsibleItemsMultiSelect.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { all, propEq, any } from 'ramda';
+
+import { Props, propTypes, Option } from './types';
+export { Props, Option } from './types';
+import NxCheckbox from '../../NxCheckbox/NxCheckbox';
+import MultiSelectCounter from './MultiSelectCounter';
+import AbstractCollapsibleItemsSelect, { generateId } from '../AbstractCollapsibleItemsSelect';
+
+function NxCollapsibleItemsMultiSelect<T extends Option>(props: Props<T>) {
```

Comment:
Could be shortened to `NxCollapsibleMultiSelect` IMO.  And similarly with the radio one